tests/queries/0_stateless/01731_async_task_queue_wait.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+# regression for 'Empty task was returned from async task queue' during query
+# cancellation with async_socket_for_remote=1 (that ignores
+# max_distributed_connections)
+$(timeout --signal=SIGINT 1 clickhouse client --max_distributed_connections=1 --max_block_size=2  --interactive_delay=900000 -q "select x  from remote('127.{2,3}', view(select number + sleep(0.3) as x from numbers(16))) settings max_block_size = 2") 2>&1 | grep "Empty task was returned from async task queue"
```

Comment:
@KochetovNicolai  don't you think that 1 second may be not enough in debug build for clickhouse-client to start the query?

Also I guess it is better to use `$CLICKHOUSE_CLIENT`

Reply:
I think it is ok if this query may fail at least in release build. Even if it happens sometimes.

Reply:
Yes, I need to use CLICKHOUSE_CLIENT